### PR TITLE
fix(auth): refresh token should be array #131

### LINKED ISSUE #131: Refresh token should be array
### Describe the bug

In the user model (src/models/apps/auth/user.models.js) 
the refresh token is string that means and on every login we are overriding the refresh token, that means if a use has logged in into 2 devices then in the user model the refresh token will be of last logged in device, so when the access token of 1st device expired and user sent a request for generating new access token with refresh token, it will not match with the refresh token stored in the user model as it has been override by the 2nd device login. 
Similarly when a use log out we are removing the refresh token means, if a  user is logged into multiple devices and in one of the device he logged out then he can't re generate the access token in any of the other devices as well because the refresh token is removed from the user model

### To Reproduce

_No response_

### Expected behavior

_No response_

### Screenshots

_No response_

### OS

_No response_

### OS Version

_No response_

### Client

_No response_

### Additional context or Information

_No response_

### PR DESCRIPTION
**Pull request by Youngemmy5956**
fix #131

### CODE REVIEW

**Review by Youngemmy5956 — COMMENTED** [review]
@hiteshchoudhary  Please checkout my pullrequest

File: .gitignore [hunk 1]
```diff
@@ -152,3 +152,7 @@ NOTES.md
 /playwright-report/
 /blob-report/
 /playwright/.cache/
+
+
+.env
+/node_modules
```
Comment (jwala-anirudh) [inline .gitignore:158]: Invalid, these are already present

**Review by Youngemmy5956 — COMMENTED** [review]
The gitignore has ben adjusted

**Comment by wajeshubham** [comment]
There is no point converting refresh token from String to Array. Anyways we are keeping latest refresh token in the field so keeping it array gives an extra overhead of pushing and pulling rather than just replacing the token.

**Pull request closed without merging**